docs/src/specification/encoding.rst

```diff
@@ -5,30 +5,42 @@ Most repo commands and status reports are Data packets whose Content contains
 ``RepoCommandParam`` or ``RepoCommandRes`` structure.
 These Data are issued via Pub-Sub protocol.
 Each ``RepoCommandParam`` and ``RepoCommandRes`` contains
-multiple ``ObjectParam`` and ``ObjectResult``, resp.
+multiple ``ObjParam`` and ``ObjStatus``, resp.
 These structures are defined as follows:
 
 .. code-block:: abnf
 
-    ObjectParam =
+    ObjParam =
         Name
         [ForwardingHint]
         [StartBlockId]
         [EndBlockId]
         [RegisterPrefix]
 
-    ObjectResult =
+    SyncParam =
+        SyncPrefix
+        [RegisterPrefix]
+        [DataNameDedupe]
+        [Reset]
+
+    ObjStatus =
         Name
         StatusCode
         [InsertNum]
         [DeleteNum]
 
+    SyncStatus =
+        Name
+        StatusCode
+
     RepoCommandParam =
```

Comment:
The **insert** topic only accepts RepoCommandParam that contains ObjParam.
The **sync/join** topic only accepts RepoCommandParam that contains SyncParam.
This suggests that these should be two separate types: ObjCommandParam and SyncCommandParam.

Reply:
I agree. We really should separate them.